solver/src/driver/solver_settlements.rs

```diff
@@ -27,14 +27,25 @@ pub fn filter_bad_settlements(
 #[derive(Debug)]
 pub struct SolverWithSettlements {
     pub name: &'static str,
-    pub settlements: Vec<RatedSettlement>,
+    pub settlements: Vec<Settlement>,
 }
 
 // Each individual settlement has an objective value.
 #[derive(Debug)]
 pub struct RatedSettlement {
     pub settlement: Settlement,
-    pub objective_value: BigRational,
+    pub surplus: BigRational,
+    pub gas_estimate: BigRational,
```

Comment:
nano-nit: I'd find it slightly more natural to be the integer type and only convert it to Rational when needed.